mmv1/third_party/terraform/services/datastream/resource_datastream_stream_test.go

```diff
@@ -63,12 +63,22 @@ func TestAccDatastreamStream_update(t *testing.T) {
 				ResourceName:            "google_datastream_stream.default",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"create_without_validation", "stream_id", "location", "desired_state", "labels", "terraform_labels"},
+				ImportStateVerifyIgnore: []string{"gtid", "binaryLogPosition", "create_without_validation", "stream_id", "location", "desired_state", "labels", "terraform_labels"},
 			},
 			{
 				// Disable prevent_destroy
 				Config: testAccDatastreamStream_datastreamStreamBasicUpdate(context, "RUNNING", false),
 			},
```

Comment:
We need setting prevent destroy to false to be the last step in this test. Otherwise this resource cannot be cleaned up

Reply:
Done.